### PR TITLE
Update `dotnet run` Documentation

### LINKED ISSUE #15521: Review `dotnet run` documentation
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-run

### PR DESCRIPTION
**Pull request by sfoslund**
## Summary

Adding option descriptions for 3.0

Fixes #15521

### CODE REVIEW

File: docs/core/tools/dotnet-run.md [hunk 1]
```diff
@@ -62,7 +70,66 @@ To run the application, the `dotnet run` command resolves the dependencies of th
 
 ## Options
 
-# [.NET Core 2.1](#tab/netcore21)
+# [.NET Core 3.0 and 3.1](#tab/netcore21)
```
Comment (mairaw) [inline docs/core/tools/dotnet-run.md]:
```suggestion
# [.NET Core 3.0 and 3.1](#tab/netcore30)
```

File: docs/core/tools/dotnet-run.md [hunk 2]
```diff
@@ -15,7 +15,15 @@ ms.date: 05/29/2018
 
 <!-- markdownlint-disable MD025 -->
 
-# [.NET Core 2.1](#tab/netcore21)
+# [.NET Core 3.0 and 3.1](#tab/netcore21)
```
Comment (mairaw) [inline docs/core/tools/dotnet-run.md]:
```suggestion
# [.NET Core 3.0 and 3.1](#tab/netcore30)
```

File: docs/core/tools/dotnet-run.md [hunk 3]
```diff
@@ -62,7 +70,66 @@ To run the application, the `dotnet run` command resolves the dependencies of th
...
+
+Forces all dependencies to be resolved even if the last restore was successful. Specifying this flag is the same as deleting the *project.assets.json* file.
+
+`-h|--help`
+
+Prints out a short help for the command.
+
+`--interactive`
+
+Allows the command to stop and wait for user input or action (for example to complete authentication).
```
Comment (mairaw) [inline docs/core/tools/dotnet-run.md]:
```suggestion
Allows the command to stop and wait for user input or action (for example, to complete authentication).
```

File: docs/core/tools/dotnet-run.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: dotnet run command
 description: The dotnet run command provides a convenient option to run your application from the source code.
-ms.date: 05/29/2018
+ms.date: 10/31/2019
 ---
 # dotnet run
```
Comment (mairaw) [inline docs/core/tools/dotnet-run.md]: Replace line 8 with the following markdown:

```markdown
**This article applies to: ✓** .NET Core 1.x SDK and later versions

<!-- todo: uncomment when all CLI commands are reviewed
[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
-->
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @sfoslund with your help with this. I've been making some formatting changes and getting rid of the tabs as I'm updating these docs, but I can do this in a separate PR (e.g. https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build).

**Review by dsplaisted — APPROVED** [review]

**Review by mairaw — APPROVED** [review]
Looks good. Thanks @sfoslund!

**Pull request merged**